packages/cli/src/cli-default-servient.ts

```diff
@@ -70,7 +70,7 @@ export default class DefaultServient extends Servient {
         },
         http: {
             port: 8080,
-            selfSigned: false,
+            allowSelfSigned: false,
```

Comment:
While I was at it, I corrected the key used here to actually correspond with the `HttpConfig` interface.